### PR TITLE
homework for chapter 1 and 2

### PR DESCRIPTION
**Pull request by wajeht**
- read in age, name, weight, height, from keyboard
- calculate bmi

### CODE REVIEW

File: Src/BootCamp.Chapter/Program.cs [hunk 1]
```diff
@@ -6,7 +6,33 @@ class Program
...
+            string sureName = "";
+            int age = 0;
+            float weight = 0f;
+            float height = 0f;
+
+            Console.Write("Name: ");
+            name = Console.ReadLine();
+
+            Console.Write("Surename: ");
+            sureName = Console.ReadLine();
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Remove this useless assignment to local variable 'sureName'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4864232)

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4864232&bid=16132684)

File: Src/BootCamp.Chapter/Program.cs (hunk 1 above)
Comment (wajeht) [reply Src/BootCamp.Chapter/Program.cs]: Done!

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4864232&bid=16132684)

File: Src/BootCamp.Chapter/Program.cs [hunk 2]
```diff
@@ -6,7 +6,33 @@ class Program
...
+            string name = "";
+            //string sureName = "";
+            int age = 0;
+            float weight = 0f;
+            float height = 0f;
+
+            Console.Write("Name: ");
+            name = Console.ReadLine();
+
+            //Console.Write("Surename: ");
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4864232)

File: Src/BootCamp.Chapter/Program.cs [hunk 3]
```diff
@@ -6,7 +6,33 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            string name = "";
+            //string sureName = "";
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4864232)

File: Src/BootCamp.Chapter/Program.cs (hunk 2 above)
Comment (wajeht) [reply Src/BootCamp.Chapter/Program.cs]: okay, done!

**Comment by Almantask** [comment]
Could declare local variables closer to first usage, but this is good enough.

**Comment by wajeht** [comment]
all done

**Comment by ellmaster** [comment]
@wajeht 
You still need to read in a second person's data! :) Please fix.

**Comment by Almantask** [comment]
Expired

**Pull request closed without merging**